.dependabot/config.yml

```
@@ -0,0 +1,10 @@
+version: 1
+update_configs:
+  - package_manager: "go:modules"
+    directory: "/"
+    update_schedule: "daily"
+    ignored_updates:
+      - match:
+          # Ignore k8s.io dependencies
+          dependency_name: "k8s.io/*"
```

```suggestion
          dependency_name: "k8s.io/*"
          version_requirement: ">=0.18.0"
```
This will allow compatible patch updates but ignore any major/minor bump